.cirrus.yml

```diff
@@ -94,10 +94,29 @@ build_task:
     - git checkout
     - cd -
     - source cirrus-env BUILD
-    - regular_mvn_build_deploy_analyze -DfailStubGenerationFast=true
+    - regular_mvn_build_deploy_analyze -DfailStubGenerationFast=true -DskipTypshed -DskipTests=true -Dsonar.skip=true
```

Comment:
If we're not executing the tests, could we even go one step further and not initialize the submodules?
Or do we still expect things to fail here in case of checksum mismatch?

Reply:
I think you're right, we don't need it here. I moved it down to the analyze_task, where we probably do need it.